public/index.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
And again: this suggests that this repository is about some Laravel project. It's not, because it's a library. Maybe look here: https://github.com/laravel/framework There's no public/index.php anywhere in plain sight. 

If you want to have a way to manually run a website, please move it to `tests/manual` or something like that.